.github/ISSUE_TEMPLATE/documentation_request.yml

```diff
@@ -0,0 +1,49 @@
+name: Documentation request
+description: Request changes or additions to documentation
+title: "[DOC] "
+labels: [documentation]
```

Comment:
name: Documentation Request 🧑‍💻
title: "[DOC 🧑‍💻]: "
labels: ["Up-for-Grabs ✋", "documentation 📃"]